src/web_app/chrome_app/manifest.json

```diff
@@ -5,7 +5,7 @@
   "manifest_version": 2,
   "app": {
     "background": {
-      "scripts": ["js/background.js"]
+      "scripts": ["js/apprtc.debug.js", "js/background.js"]
```

Comment:
Well. As opposed to loading the same code twice? Because now everything is loaded in two pages when not all of it has to be. Seems to increase memory footprint unnecessarily. Don't have to fix it now, because I think either way package size / memory size is small. But it just felt the same like having a bunch of header files in an impl file that aren't actually being used, especially since they all pollute the global namespace currently.